### ISSUE TITLE
Disabled input has poor colour contrast

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#31207]
Disabled input component, e.g. at `/me/account` page has poor colour contrast:

![image](https://user-images.githubusercontent.com/87168/53807870-6c1d2780-3f59-11e9-9120-478724a79ef9.png)

https://webaim.org/resources/contrastchecker/?fcolor=B0B5B8&bcolor=F6F6F6

![image](https://user-images.githubusercontent.com/87168/53807878-73443580-3f59-11e9-88c8-75de36df094c.png)

cc @drw158 @flootr

**Comment by simison** [comment]
Similarly for disabled button:

![image](https://user-images.githubusercontent.com/87168/53807970-b6060d80-3f59-11e9-95f5-2ce62904fd32.png)


but that doesn't really matter as much as the username, which I really should be able to read. 

Quick "fix" would be to not have it as an input at all?

**Comment by torres126** [comment]
Noting that #26982 has discussion of this.

**Comment by drw158** [comment]
> Quick "fix" would be to not have it as an input at all?

Does it ever need to be an input?

**Comment by torres126** [comment]
Yes: https://en.support.wordpress.com/change-your-username/

But I can't reproduce that state, am I missing something? 

![Screenshot_20190317-072256](https://user-images.githubusercontent.com/43215253/54486829-a229a800-4885-11e9-9c74-825bfcc5a83a.jpg)

**Comment by simison** [comment]
>> Quick "fix" would be to not have it as an input at all?
>
> Does it ever need to be an input? Is there ever a case where you can change your username?

Yeah, good point, it can be changed. :-/

>But I can't reproduce that state, am I missing something?

I get disabled input-state because I have `user_login_can_be_changed` user setting; this is a small, specific subset of users so probably for this page this is a non-problem.

https://github.com/Automattic/wp-calypso/blob/59e0d99efde75065e6f314f25aab54ed35c0ad09/client/me/account/main.jsx#L800-L802

I think you'd also get disabled state when you'd start actually changing the username and input is disabled between loading/validation and such.

I'll close this one as it's about this specific input and https://github.com/Automattic/wp-calypso/issues/26982 is a good generic one about disabled states for form elements.

**Comment by simison** [comment]
Another always disabled button is WordPress version input for Jetpack-connected sites:

<img width="213" alt="image" src="https://user-images.githubusercontent.com/87168/54517146-8dbedb80-4969-11e9-9725-9f41a6c266e1.png">
